models/reviews.models.js

```diff
@@ -53,6 +64,27 @@ exports.selectReview = (review_id) => {
     });
 };
 
+exports.selectReviewComments = (review_id) => {
+  if (isNaN(+review_id)) {
+    return idNotNumber();
+  }
+  return Promise.all([
+    db.query(
+      `
+  SELECT * FROM comments
+  WHERE review_id = $1
+  `,
+      [review_id]
+    ),
+    checkReviewIDExists(review_id),
+  ]).then(([result]) => {
+    const comments = result.rows;
+    if (!comments) {
+      return idNotFound();
+    } else return comments;
```

Comment:
By the time you've got the result you've already checked that the review_id exists. Your if statement will always be false here as comments is an array.

You can remove the if entirely and return the comments in all cases